astropy/extern/ply/lex.py

```diff
@@ -41,8 +41,6 @@
 import os
 import inspect
 
-from ...extern.six.moves import zip
```

Comment:
Still, I would rather follow the policy to keep the external packages untouched here, but it would be great if you could contribute any improvements upstream.